samples/asgardeo-choreo-react-express/apps/server/package.json

```diff
@@ -5,10 +5,10 @@
   "type": "module",
   "author": "Yathindra Kodithuwakku <yathindrarawya123@gmail.com>",
```

Comment:
Shall we update the author to "WSO2" with this PR, if possible?

Reply:
Sure